tests/evaluate/test_default_evaluator.py

```diff
@@ -633,16 +627,20 @@ def test_pipeline_model_kernel_explainer_on_categorical_features(pipeline_model_
             evaluators="default",
             evaluator_config={"explainability_algorithm": "kernel"},
         )
-    run_data = get_run_data(run.info.run_id)
+    run_id = run.info.run_id
+    run_data = get_run_data(run_id)
     assert {
         # TODO: Uncomment once https://github.com/shap/shap/issues/3901 is fixed
         # "shap_beeswarm_plot.png",
         "shap_feature_importance_plot.png",
         "shap_summary_plot.png",
-        "explainer",
     }.issubset(run_data.artifacts)
 
-    explainer = mlflow.shap.load_explainer(f"runs:/{run.info.run_id}/explainer")
+    # TODO: add `and name='explainer'` once sqlAlchemyStore search_logged_models supports it
+    model = mlflow.search_logged_models(
```

Comment:
It'd be nice if we had a `get_logged_model_by_name` that fetches the model matching the name.

Reply:
Yes seems so, but name is not unique?

Reply:
Yeh I think we can fetch the latest model matching the name.